node_lambnik/scripts/updatedb

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+set -e
+
+# Execute psql commands on the database container for performance reasons.
+CONTAINER_ID=$(docker-compose -f docker-compose.yml ps -q database)
+PSQL="docker exec -i $CONTAINER_ID gosu postgres psql -d lambnik-test"
+# PSQL="psql -d lambnik_test -h rds-lambnik-test.cpz8rmcptx2k.us-east-1.rds.amazonaws.com -U lambnik -W"
+
+function usage() {
+    echo -n \
+         "Usage: $(basename "$0") [-h] [-f]
+
+Populate database.
+
+Options:
+    -h  Show this help text
+    -f  Force data reload
+"
+}
+
+function load() {
+    file=${1}
+    extracted_file=${file/.gz/}
+    gunzip "$file" -qkf
+    ${PSQL} < "$extracted_file"
+    rm "$extracted_file"
+}
+
+function table_exists() {
+    table=${1}
+    if [ "$force_reload" = true ]; then
+        echo "Table $table will be created or reloaded"
+        return 1
+    else
+        ${PSQL} -c "SELECT 1 FROM $table LIMIT 1" &>/dev/null
+    fi
+}
+
+# Source: https://github.com/azavea/raster-foundry/blob/develop/scripts/setup#L17
+function check_database() {
+    # Check if database is set up to continue
+    max=21 # 1 minute
+    counter=1
+    while true; do
+        if [[ ${counter} -gt 1 ]]; then
+            echo "Checking if database is up yet (attempt ${counter})..."
+        fi
+        set +e
+        ${PSQL} -c "SELECT 1" &>/dev/null
+        status_check=$?
+        if [ $status_check == 0 ]; then
+            return
+        fi
+        set -e
+        if [[ ${counter} == "${max}" ]]; then
+            echo "Could not connect to database after some time"
+            exit 1
+        fi
+        sleep 3
+        (( counter++ ))
+    done
+}
+
+function check_postgis_install() {
+    ${PSQL} -c "SELECT PostGIS_full_version();" &>/dev/null
+    status_check=$?
+    if [ $status_check == 0 ]; then
+        echo "PostGIS is enabled"
+        return
+    fi
+    echo "Enabling PostGIS"
+    ${PSQL} -c "CREATE EXTENSION postgis;" &>/dev/null
+}
+
+while getopts ":hf" opt; do
+    case $opt in
+        h) usage; exit 0;;
+        f) force_reload=true ;;
+        \?) usage; exit 1;;
+    esac
+done
+
+check_database;
+check_postgis_install;
+
+if table_exists "pwd_inlets"; then
```

Comment:
This is producing false positives for me, so it's skipping the import.  I believe I ran into the same problem working on setting up the database for `breakable_toy`, so the [load_db.sh](https://github.com/azavea/tilegarden/blob/develop/breakable_toy/scripts/load_db.sh) script in there is a little more robust, and should be easy to adapt (maybe just changing the default tablename).

Reply:
I'm on docker-compose 1.17.1 as well. Those publish commands do work the same, I'll get rid of the bash part.